tests/bearlib/languages/documentation/DocstyleDefinitionTest.py

```diff
@@ -73,7 +92,13 @@ def test_load(self):
         self.assertEqual(result.docstyle, "default")
         self.assertEqual(result.markers, (('"""', '', '"""'),))
 
+        self.assertEqual(result.metadata, self.dummy_metadata)
+        self.assertEqual(result.metadata.param_start, ":param ")
+        self.assertEqual(result.metadata.param_end, ":")
+        self.assertEqual(result.metadata.return_sep, ":return:")
+
     def test_load_external_coalang(self):
+        empty_metadata = self.Metadata('', '', '')
```

Comment:
not for this PR, but maybe we can check for this case on docstyle load and use `None` for invalid / missing metadata definitions :+1: